### ISSUE TITLE
Accessibility: Missing title on receipt page

### ISSUE THREAD

**Issue by nicolaisweb** [globaleaks/GlobaLeaks#3434]
### What version of GlobaLeaks are you using?

4.11.5

### What operating system(s) are you seeing the problem on?

Windows, macOS, Android, iOS, Linux

### Describe the issue

On the final page of the questionnaire, the title is missing.

### Proposed solution

Enclose all the generated text (currently inside the div with id "ReceiptText") inside an h1.
Remove the "span" tags.

**Actual Code**

        <div id="ReceiptText">
          <span data-translate="">Grazie.</span>
          <span data-translate="">La tua segnalazione è andata a buon fine.</span>
          <span data-translate="">Cercheremo di risponderti quanto prima.</span>
        </div>

**Suggested code**

        <h1 id="ReceiptText">
          Grazie. La tua segnalazione è andata a buon fine. Cercheremo di risponderti quanto prima.
        </h1>

**Comment by evilaliv3** [comment]
Than you @nicolaisweb 

Would you please clarify where do you read that any page requires an H1 element?

As far that we know any page require a <title> tag that is actually always present.

**Comment by nicolaisweb** [comment]
Certainly, below are some useful links:

[https://fae.disability.illinois.edu/rulesets/HEADING_1/](url)
[https://www.w3.org/WAI/tutorials/page-structure/headings/](url)
